src/services/codefixes/convertConstToLet.ts

```diff
@@ -11,7 +11,7 @@ namespace ts.codefix {
             if (range === undefined) return;
 
             const changes = textChanges.ChangeTracker.with(context, t => doChange(t, sourceFile, range));
-            return [createCodeFixAction(fixId, changes, Diagnostics.Convert_const_to_let, fixId, Diagnostics.Convert_const_to_let)];
+            return [createCodeFixActionWithoutFixAll(fixId, changes, Diagnostics.Convert_const_to_let)];

```

Comment:
Does this mean that something like

```ts
const a = 1;
const b = 1;

a = 2;
b = 2;
```

will no longer get a fix-all for both `a` and `b`?

Reply:
@DanielRosenwasser The current implementation doesn't support `Fix All`, it uses `createCodeFixAction`, however, there is no `getAllCodeActions` implementation, as a result, we get `Fix All` action which does absolutely nothing. So I guess instead of using `createCodeFixAction` we should use `createCodeFixActionWithoutFixAll`.

--- 
It would be nice to have a `Fix All` action, however, there are a couple of questions

```ts
const a = 1;
a = 1; // Convert const to let 
a = 1; // Convert const to let 
```

We get two similar errors, do we need to have a `Fix All` action?

So it looks like we need to group by symbol id, however, which node should have the `Fix All` action?

```ts
const a = 1;
a = 1; // Convert const to let ? Fix All
a = 1; // Convert const to let ? Fix All

const b = 1;
b = 1; // Convert const to let ? Fix All
b = 1; // Convert const to let ? Fix All
```

Reply:
I think this is similar to the issue on missing object-like constraints (which @weswigham and you have each worked on a bit).

Are we limited to having a single fix-all on each node? Is it bad to have the same fix-all on each of these?